docs/docs/advanced/useComposedEventHandler.mdx

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 6
+---
+
+# useComposedEventHandler
+
+This is a hook that lets you compose [useEvent](/docs/advanced/useEvent)-based event handlers (such as [useAnimatedScrollHandler](/docs/scroll/useAnimatedScrollHandler) or your own custom ones) into a single, combined event handler.
+
+### Arguments
+
+#### `handlers`
+
+An array of event handlers created using [useEvent](/docs/advanced/useEvent) hook. `useComposedEventHandler` hook reacts to any changes in given handlers and rebuilds whenever it is needed.
+
+### Returns
+
+The hook returns a handler object that can be hooked into any [`Animated component`](/docs/fundamentals/glossary#animated-component).
+The handler should be passed to a corresponding `onEvent` prop (e.g. `onScroll` when working with scroll-related handlers). In reality, the composed handler can be hooked into virtually any prop thanks to the way it is detected. Nevertheless, we strongly recommend setting the handler on the prop it belongs to.
```

Comment:
> In reality, the composed handler can be hooked into virtually any prop thanks to the way it is detected. Nevertheless, we strongly recommend setting the handler on the prop it belongs to.

I think we should rephrase that so we don't encourage bad habits. Probably into something like:


> If your composed handler aggregates multiple events that have little in common, put it into a new property, e.g. "composedHandler". However, the best practice here would be to use several composed handlers for code clarity.

Or something among these lines.